### PR TITLE
docs: Correct some examples of config file

### LINKED ISSUE #210: Update docs for chore: Rename MetaClientOpts to MetaClientOptions
A document change request is generated from
https://github.com/GreptimeTeam/greptimedb/pull/1075

### PR DESCRIPTION
**Pull request by evenyag**
This PR corrects outdated examples in the configuration section.

This fixes #210

### CODE REVIEW

File: docs/reference/configuration.md [hunk 1]
```diff
@@ -125,8 +141,18 @@ When you use GreptimeDB in the standalone mode, you can configure it as below:
...
-http_addr = '127.0.0.1:4000'
-wal_dir = "/tmp/greptimedb/wal/"
+
+[http_options]
+addr = '127.0.0.1:4000'
+timeout = "30s"
+
+[wal]
+dir = "/tmp/greptimedb/wal"
+file_size = '1GB'
```
Comment (xtang) [inline docs/reference/configuration.md:151]:
```suggestion
file_size = "1GB"
```

File: docs/reference/configuration.md [hunk 2]
```diff
@@ -125,8 +141,18 @@ When you use GreptimeDB in the standalone mode, you can configure it as below:
...
+purge_interval = '10m'
```
Comment (xtang) [inline docs/reference/configuration.md:152]:
```suggestion
purge_interval = "10m"
```

File: docs/reference/configuration.md [hunk 3]
```diff
@@ -125,8 +141,18 @@ When you use GreptimeDB in the standalone mode, you can configure it as below:
...
+purge_threshold = '50GB'
```
Comment (xtang) [inline docs/reference/configuration.md:153]:
```suggestion
purge_threshold = "50GB"
```

File: docs/reference/configuration.md [hunk 4]
```diff
@@ -20,29 +20,44 @@ greptime [standalone | frontend | datanode | metasrv]  start -c config/standalon
 Common protocol configurations in `frontend` and `standalone` sub command:
 
 ```toml
+[http_options]
+addr = "127.0.0.1:4000"
+timeout = "30s"
+
 [grpc_options]
-addr = '127.0.0.1:4001'
+addr = "127.0.0.1:4001"
```
Comment (xtang) [inline docs/reference/configuration.md:28]: I noticed that there are still single quotation marks in the latter parts of the file. Would it be better to use a consistent type of quotation mark throughout the entire configuration file?

Comment (evenyag) [reply docs/reference/configuration.md:28]: Okay, I will address it.

File: docs/reference/configuration.md [hunk 5]
```diff
@@ -152,38 +175,35 @@ runtime_size = 2
...
 metasrv_addrs = ["127.0.0.1:3002"]
 timeout_millis = 3000
 connect_timeout_millis = 5000
 tcp_nodelay = false
 ```
 
-The `datanode_rpc_addr` is not used in `distributed` mode, you can leave it in default value.
-
-The `meta_client_opts` configure the metasrv client confugrations, incuding:
+The `meta_client_options` configure the metasrv client configuration, incuding:
```
Comment (xtang) [inline docs/reference/configuration.md:201]:
```suggestion
The `meta_client_options` configure the metasrv client, including:
```

File: docs/reference/configuration.md [hunk 6]
```diff
@@ -217,15 +244,15 @@ Datanode in distributed mode should set different `node_id` in different nodes.
...
 datanode_lease_secs = 30
 ```
 
 | Key                 | Type    | Description                                                                                                             |
 |---------------------|---------|-------------------------------------------------------------------------------------------------------------------------|
-| bind_addr           | String  | The bind address of metasrv, '127.0.0.1:3002' by default.                                                                 |
-| server_addr         | String  | The communication server address for frontend and datanode to connect metasrv,  '127.0.0.1:3002' by default for localhost |
-| store_addr          | String  | Etcd server address, '127.0.0.1:2379' by default                                                                        |
+| bind_addr           | String  | The bind address of metasrv, "127.0.0.1:3002" by default.                                                                 |
+| server_addr         | String  | The communication server address for frontend and datanode to connect metasrv,  "127.0.0.1:3002" by default for localhost |
```
Comment (xtang) [inline docs/reference/configuration.md:256]:
```suggestion
| server_addr         | String  | The communication server address for frontend and datanode to connect to metasrv,  "127.0.0.1:3002" by default for localhost |
```

File: docs/reference/configuration.md [hunk 7]
```diff
@@ -52,48 +67,47 @@ All of these options are optional, the default values are listed above. If you w
...
 
 
-| Option | Key       | Type    | Description                                                                        |
-|--------|-----------|---------|------------------------------------------------------------------------------------|
-|        | node_id   | Integer | The datanode identifier, set it 0 in standalone mode, otherwise should be different|
-|        | mode      | String  | Node running mode, includes 'standalone' or 'distributed'                          |
-|        | http_addr | String  | HTTP API server address, '127.0.0.1:4000' by default                                 |
+| Option | Key                     | Type    | Description                                                                        |
+|--------|-------------------------|---------|------------------------------------------------------------------------------------|
+|        | mode                    | String  | Node running mode, includes "standalone" or "distributed"                          |
```
Comment (evenyag) [inline docs/reference/configuration.md:97]: Should I change this to? @xtang 
```suggestion
|        | mode                    | String  | Node running mode, includes "standalone" and "distributed"                          |
```

Comment (xtang) [reply docs/reference/configuration.md]: Sounds great. It's better.

**Review by xtang — APPROVED** [review]
LGTM

**Pull request merged**